README-local-package-graph.md

```diff
@@ -0,0 +1,16 @@
+## To make a Mermaid graph for your operator index
```

Comment:
Why we need 2 README(s) file? Could we centralize all info in the README?

Reply:
This is specialized to just cover the case where a user is an operator author and wants the side instructions on how to graph their own (possibly not published) operator locally. I think 2nd README is justified because it's not meant for the general audience.